### PR TITLE
allow arbitrary maximums for pulse project lists - #1506

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Kinda weird...but `¯\_(ツ)_/¯`

For returning every result I'd suggest putting in a very large max, like 1000.

File: source/js/components/pulse-project-list/pulse-project-list.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default class PulseProjectList extends React.Component {
       });
     });
 
-    projectXHR.open(`GET`, `https://${this.props.env.PULSE_API_DOMAIN}/api/pulse/v2/entries/?format=json&search=${query}${this.props.featured ? `&featured=True` : ``}`);
+    projectXHR.open(`GET`, `https://${this.props.env.PULSE_API_DOMAIN}/api/pulse/v2/entries/?format=json&page_size=${this.props.max ? this.props.max : 48}&search=${query}${this.props.featured ? `&featured=True` : ``}`);
```
Comment (alanmoo) [inline source/js/components/pulse-project-list/pulse-project-list.jsx]: So 0 gets evaluated as false in the ternary and thus fetches 48?

Comment (gvn) [reply source/js/components/pulse-project-list/pulse-project-list.jsx]: Correctamundo.

File: network-api/networkapi/wagtailpages/customblocks.py [hunk 2]
```diff
@@ -267,7 +267,7 @@ class PulseProjectList(blocks.StructBlock):
         min_value=0,
         default=0,
         required=False,
-        help_text='The maximum number of results to fetch (use 0 for "no maximum")',
+        help_text='The maximum number of results to fetch (use 0 for default maximum of 48)',
```
Comment (alanmoo) [inline network-api/networkapi/wagtailpages/customblocks.py]: Believe it or not, this needs a migration, thus the CI failure

Comment (gvn) [reply network-api/networkapi/wagtailpages/customblocks.py]: Just noticed that...hang on

**Pull request merged**